posthog/event_usage.py

```diff
@@ -144,17 +162,48 @@ def report_bulk_invited(
             "current_member_count": current_member_count,
             "email_available": email_available,
         },
+        groups=groups(user.current_organization, user.current_team),
     )
 
 
-def report_org_usage(distinct_id: str, properties: Dict[str, Any]) -> None:
+def report_org_usage(organization_id: str, distinct_id: str, properties: Dict[str, Any]) -> None:
     """
     Triggered daily by Celery scheduler.
     """
     posthoganalytics.capture(
-        distinct_id, "organization usage report", properties,
+        distinct_id,
+        "organization usage report",
+        properties,
+        groups={"organization": organization_id, "instance": SITE_URL},
+    )
+    posthoganalytics.group_identify("organization", organization_id, properties)
```

Comment:
What's the goal of this here? We don't run any additional capture calls afterwards, right? Is this saving any other attributes?

Reply:
I think this sets organization attributes, thus behaving like `$set`. Correct @macobo?

BTW, I'm happy to help write documentation for group APIs whenever we feel the time is right.

Reply:
This is indeed updating organization attributes. There already is some (hidden) docs over at https://posthog.com/docs/user-guides/group-analytics and we're finishing up the full documentation this sprint :)